### PR TITLE
hosted: Misc cherry-picks

### PR DESCRIPTION
**Pull request by lluiscampos**
Namely #1669, #1676, #1677, #1683 and #1685.

### CODE REVIEW

File: 09.Downloads/docs.md [hunk 1]
```diff
@@ -382,57 +382,57 @@ download it by running:
...
 [/ui-tabs]
 
 
 Then install the package with:
 
-<!--AUTOVERSION: "mender-monitor_demo_%-1_all.deb"/monitor-client -->
+<!--AUTOVERSION: "mender-monitor_demo_%-1"/monitor-client -->
 ```bash
-dpkg -i mender-monitor_demo_1.1.0-1_all.deb
+dpkg -i mender-monitor_demo_1.1.0-1+debian+buster_all.deb
```
Comment (merlin-northern) [inline 09.Downloads/docs.md:437]: this package does not exist, the correct file name is:
```
mender-monitor-demo_1.1.0-1+debian+buster_all.deb
```
same for the above wgets of it.

**Review by merlin-northern — COMMENTED** [review]
one comment.

File: 09.Downloads/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:437]: Good catch! This is incorrect in master too so I'll submit the fix there first and then cherry-pick

Comment (lluiscampos) [reply 09.Downloads/docs.md]: Cherry-picked now #1683 also.

File: 09.Downloads/docs.md [hunk 2]
```diff
@@ -382,57 +382,57 @@ download it by running:
...
 [/ui-tabs]
 
 
 Then install the package with:
 
-<!--AUTOVERSION: "mender-monitor_demo_%-1_all.deb"/monitor-client -->
+<!--AUTOVERSION: "mender-monitor-demo_%-1"/monitor-client -->
 ```bash
-dpkg -i mender-monitor_demo_1.1.0-1_all.deb
+dpkg -i mender-monitor-demo_1.1.0-1+debian+buster_all.deb
```
Comment (merlin-northern) [inline 09.Downloads/docs.md:437]: behold. I went through the download:
```
curl --fail -u "..." -O https://downloads.customer.mender.io/content/hosted/mender-monitor/debian/1.1.0/mender-monitor-demo_1.1.0-1%2Bdebian%2Bbuster_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3372  100  3372    0     0   1885      0  0:00:01  0:00:01 --:--:--  1885
#ls mender-monitor-demo*
mender-monitor-demo_1.1.0-1%2Bdebian%2Bbuster_all.deb
```
https://github.com/curl/curl/issues/2573

Comment (merlin-northern) [reply 09.Downloads/docs.md:437]: we have to use wget:
```
wget --auth-no-challenge  --user=username --password=password https://downloads.customer.mender.io/content/hosted/mender-monitor/debian/1.1.0/mender-monitor-demo_1.1.0-1%2Bdebian%2Bbuster_all.deb
mender-monitor/debian/1.1.0/mender-monitor-demo_1.1.0-1%2Bdebian%2Bbuster_all.deb
--2022-02-17 11:06:29--  https://downloads.customer.mender.io/content/hosted/mender-monitor/debian/1.1.0/mender-monitor-demo_1.1.0-1%2Bdebian%2Bbuster_all.deb
Resolving downloads.customer.mender.io (downloads.customer.mender.io)... 3.232.69.107, 34.192.90.45
Connecting to downloads.customer.mender.io (downloads.customer.mender.io)|3.232.69.107|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 3372 (3.3K) [application/x-debian-package]
Saving to: 'mender-monitor-demo_1.1.0-1+debian+buster_all.deb'

mender-monitor-demo_1.1.0-1+debian+ 100%[==================================================================>]   3.29K  --.-KB/s    in 0s

2022-02-17 11:06:31 (13.4 MB/s) - 'mender-monitor-demo_1.1.0-1+debian+buster_all.deb' saved [3372/3372]
```

Comment (lluiscampos) [reply 09.Downloads/docs.md:437]: Okey... I'll submit a fix in master and cherry-pick again everywhere :stuck_out_tongue:

**Pull request merged**